### PR TITLE
Rating grid

### PR DESCRIPTION
**Pull request by Sudhanshusoni45**
### Changes to check:

1.Added rating component.
2.Added grid component.

[Preview Link](https://deploy-preview-8--maya-ui.netlify.app/pages/rating/rating)

### CODE REVIEW

**Review by NamrataRaikwar2002 — APPROVED** [review]
Great work sudhanshu, UI  looks nice.

File: Pages/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  src="https://carbon.now.sh/embed?bg=rgba%28156%2C195%2C249%2C1%29&t=seti&wt=none&l=htmlmixed&width=456&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid%2520grid-col-2%2520gap-1%2522%253E%250A%253Cdiv%253E1%253C%252Fdiv%253E%250A%253Cdiv%253E2%253C%252Fdiv%253E%250A%2520%2520...%250A%253Cdiv%253E6%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 100%; height: 281px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+        </div>
+
+        <div class="divider"></div>
+
+      
+
```
Comment (shristabaruah) [inline Pages/Grid/grid.html:105]: you can remove this extra spaces it increases the code of lines.

**Review by GauravDe-git — APPROVED** [review]
- Clean and well-formatted code.
- Grid component looks great! 🎊

**Review by shristabaruah — APPROVED** [review]
- Clean code
- UI looks awesome 
great work @Sudhanshusoni45

File: Pages/Grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        </div>
+
+        <div class="divider"></div>
+
+        <div class="content-head">
+          <h2>Basic grid</h2>
+        </div>
+        <div class="content">
+          <p>
+            Simple rating can be created by adding class name 'rating-container' to star icons
```
Comment (sumeetyeola) [inline Pages/Grid/grid.html:77]: This line describes rating component & not grid .

**Review by sumeetyeola — CHANGES_REQUESTED** [review]
Hey, good work @Sudhanshusoni45 with grid and rating components! Suggested a small change. Also, just a reminder to create temp-dev branch to merge this PR.

**Pull request merged**